src/auth/refreshclient.ts

```diff
@@ -32,9 +38,31 @@ export class UserRefreshClient extends OAuth2Client {
    * @param {string} refreshToken The authentication refresh token.
    * @constructor
    */
-  constructor(clientId?: string, clientSecret?: string, refreshToken?: string) {
-    super(clientId, clientSecret);
-    this._refreshToken = refreshToken;
+  constructor(
+      clientId?: string, clientSecret?: string, refreshToken?: string,
+      refreshTokenEarlyMillis?: number);
```

Comment:
Lets not add this as an option here.  I mostly kept the multiple argument style signature around purely for backwards compatibility.  As we move forward, lets just use the options objects for new properties.

Reply:
Done.